fastagency/studio/models/agents/assistant.py

```diff
@@ -14,7 +14,12 @@ class AssistantAgent(AgentBaseModel):
     system_message: Annotated[
         str,
         Field(
-            description="The system message of the agent. This message is used to inform the agent about his role in the conversation"
+            description="The system message of the agent. This message is used to inform the agent about his role in the conversation",
```

Comment:
This is not very elegant. Why don't you create a subclass of Field that has an additional `tooltip` parameter in its constructor: 

```python
import pydantic

class Field(pydantic.Field):
  __init__(self, default: Any, *, description: str, tooltip: str, **kwargs: Any):
    super():__init__(
      default,
      description=description, 
      json_schema_extra={
        "metadata": {
          "tooltip_message": tooltip
                }
            }
    )
```

Then you can write something like this:
```python
    system_message: Annotated[
        str,
        Field(
          description="whatever",
          tooltip="whocares",
       )
    )
```